config/provider.js

```diff
@@ -103,7 +111,8 @@ const SCHEMA_PROVIDER = Joi.object().keys({
         .optional()
         .valid('ARM_CONTAINER', 'LINUX_CONTAINER', 'LINUX_GPU_CONTAINER')
         .default('LINUX_CONTAINER')
-        .description('CodeBuild environment type')
+        .description('CodeBuild environment type'),
+    prefix: Joi.string().description('Prefix name for build environment, suffixed with -')
```

Comment:
why do we need this ?

Reply:
to know if build is ARM container or LINUX container, currently keeping it an open configuration under provider, later we can feature it to SD specific